code-scan.js

```diff
@@ -0,0 +1,203 @@
+const { Octokit } = require("@octokit/rest");
+const core = require("@actions/core");
+const { countReset } = require("console");
+const fs = require("fs");
+const token = process.argv[2];
+const repo = process.argv[3];
+const originalOwner = process.argv[4];
+const owner = "asml-actions-validation";
+console.log(`repo name is ${repo}, original owner ${originalOwner}`);
```

Comment:
hoping that you have only kept the necessary log statements and nothing extra :D

Reply:
removed this log